knotx-it-tests/src/test/resources/content/local/customAndDefaultSymbol.html

```diff
@@ -42,7 +42,7 @@ <h2>
     </div>
   </div>
   <div class="row">
-    <script data-knotx-knots="databridge,handlebars"
+    <script data-knotx-knots="databridge,te"
```

Comment:
Can we simplify this test to one service invocation, we test databridge namespaces in module tests.